front/src/views/routes/workspace/admin/database/DatabaseTable.vue

```diff
@@ -613,6 +686,10 @@ export default {
       this.multipleAutocompleteInput = {}
       this.displayNewDialog = true
     },
+    /**
+     * Export current view according to current filters in CSV
+     * Todo: Do the same for sort!
```

Comment:
Todo: Export of the current view according to the sort if the user applies one.

Reply:
Maybe we need to create a new issue for this use case ?

Reply:
Already done